main.js

```diff
@@ -0,0 +1,128 @@
+const alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+const gallowImages = ["1.jpg", "2.jpg", "3.jpg", "4.jpg", "5.jpg", "6.jpg", "7.jpg", "8.jpg", "https://i.ebayimg.com/images/g/jQkAAOSwo-pjeWSL/s-l1600.jpg", "https://i.pinimg.com/236x/db/62/e0/db62e024c5cd86b8a14ec068b2318c85--captain-jack-sparrow-pirates-of-the-caribbean.jpg"];
+let encryptedWord = [];
+let player1 = document.getElementById("player1");
+let player2 = document.getElementById("player2");
+let wordToGuess;
+let mismatchedLetters = 0;
+let completeBoard = encrypted;
+let alphabetButtons = letters;
+let gallowsImages = images;
+let resetButton = restart;
+
+
+// for each letter in the word, a "_" will be displayed
+function completeArray(wordToGuess) {
+    for (let i = 0; i < wordToGuess.length; ++i) {
+        encryptedWord.push(" _ ");
+    }
+    displayEncryptedWord();
+}
+
+// depending on the number of lifes, completes the board and decides if the game is over or not
+function countLifes(letterMatch) {
+    if (letterMatch == 0) {
+        ++mismatchedLetters;
+        displayImages(mismatchedLetters);
+        if (mismatchedLetters == 8) {
+            mismatchedLetters = 9;
+            gameOver(mismatchedLetters);
+        }
+    }
+}
+
+// displayes the letters of the alphabet as buttons
+// adds functionality to the buttons
```

Comment:
displays* the letters of the alphabet as buttons and* adds functionality to the buttons

Please be careful with your spelling, the comments has to make sense to be useful.